app/models/agents/twitter_user_agent.rb

```diff
@@ -57,13 +51,15 @@ def default_options
         'username' => 'tectonic',
         'include_retweets' => 'true',
         'exclude_replies' => 'false',
-        'expected_update_period_in_days' => '2'
+        'expected_update_period_in_days' => '2',
+        'choose_home_time_line' => 'true'
```

Comment:
Maybe set this to `false` to keep the default behavior of the agent like it was before?